src/styles/global.css

```diff
@@ -9,31 +9,112 @@ html {
 }
 
 :root {
-  --primary-dark: #e61a6b;
-  --primary-darker: #8b0f40;
-  --primary-light: #f5a4c4;
-  --primary-bg: #fafafa;
-  --primary-bg-dark: #d9d9d9;
-  --primary-bg-blue: #0014cd;
-  --primary-white-bright: #ffffff;
-  --primary-white: #faf8f5;
-  --primary-black: #2d2d2d;
-  --primary-black-darker: #000000;
+  --Blue-500: #3840ff;
+  --Blue-700: #0014cd;
+  --Blue-300: #c3c8f6;
+  --Blue-400: #788aea;
+  --Blue-500: #3840ff;
+  --Blue-200: #bee0ff;
+  --Blue-50: #f2f3fd;
 
-  --secondary-off-white1: #f4efe8;
-  --secondary-off-white2: #ece1d3;
+  --Green-900: #024105;
+  --Green-800: #035506;
+  --Green-700: #046d08;
+  --Green-600: #058c0a;
+  --Green-500: #059a0b;
+  --Green-400: #37ae3c;
+  --Green-300: #58bb5c;
+  --Green-200: #8cd18f;
+  --Green-100: #b2e0b3;
+  --Green-50: #e6f5e7;
 
-  --primary-red-error: #b30b0b;
-  --primary-yellow-warning: #ffd02f;
+  --Red-900: #46100a;
+  --Red-700: #7c2318;
+  --Red-600: #b63729;
+  --Red-500: #f0503f;
+  --Red-300: #f3948f;
+  --Red-100: #fabbb4;
+  --Red-50: #fdedec;
 
-  --focus-color: #8b0f40;
+  --Yellow-900: #3b2800;
+  --Yellow-800: #634601;
+  --Yellow-700: #8e6703;
+  --Yellow-600: #bd8905;
+  --Yellow-500: #edad08;
+  --Yellow-400: #ffd02f;
+  --Yellow-300: #fbe186;
+  --Yellow-100: #fdefbb;
+  --Yellow-50: #fffbf6;
 
-  /* TODO: upgrade color-scheme with correct colors */
-  --secondary-red: #f0503f;
-  --text-primary: #222424;
-  --text-tertiary: #5e5e5e;
+  --Lila-900: #220548;
```

Comment:
Rename to violet :)

Reply:
Changed it to violet now

Reply:
I think i just looked in the wrong place in Figma cause i found the color Violet in the components page 😆